handsontable/src/plugins/pagination/pagination.js

```diff
@@ -7,9 +7,12 @@ import { announce } from '../../utils/a11yAnnouncer';
 import { createPaginatorStrategy } from './strategies';
 import { toSingleLine } from '../../helpers/templateLiteralTag';
 import { warn } from '../../helpers/console';
+import { getMostBottomEndPosition } from '../../core/focusCatcher/utils';
```

Comment:
Using helper functions from the other submodule is not the best idea. If you need that function, I'd recommend moving it to the global helper (`./src/helpers`), from which Pagination and FocusCatcher could use it.